src/common/components/configurationForm/index.jsx

```diff
@@ -182,7 +182,7 @@ const ConfigurationForm = () => {
             autoCorrect="off"
             autoCapitalize="none"
             min={1}
-            max={9999}
+            max={Number.MAX_SAFE_INTEGER}
```

Comment:
I am not sure, postgresql exists server, which have more, than 9999 CPUs.

Reply:
An arbitrary limit on memory forced me to fork in order to use the tool. The UI code didn't handle not having a `max` value, so I set everything to the maximum allowable value. I don't necessarily think it makes sense to add arbitrary limits, as they cause issues for users, and aren't likely to cause typos/errors as they're implemented (since 9999 is "too high" anyway).